src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.HttpContentAsyncStream.cs

```diff
@@ -304,37 +304,21 @@ internal void Run()
                 Debug.Assert(!Monitor.IsEntered(_syncObj), "Should not be invoked while holding the lock");
                 Debug.Assert(_copyTask == null, "Should only be invoked after construction or a reset");
 
-                // Start the copy and store the task to represent it
-                _copyTask = StartCopyToAsync();
+                // Start the copy and store the task to represent it. In the rare case that the synchronous 
```

Comment:
I actually meant "synchronous" not "asynchronous".  It's the synchronous invocation of CopyToAsync that's the problem.